gef-ui/server/controllers/unissued-facilities/validation.js

```diff
@@ -84,9 +84,10 @@ const facilityValidation = async (body, query, params) => {
       subFieldErrorRefs: dateFieldsInError,
     });
   } else if (issueDateIsFullyComplete) {
-    const submissionDate = new Date(+application.submissionDate);
-    const threeMonthsFromSubmission = add(submissionDate, { months: 3 });
-    const issueDateSet = set(new Date(), { year: issueDateYear, month: issueDateMonth - 1, date: issueDateDay });
+    // set to midnight to stop mismatch if submission date in past so set to midnight of past date
+    const submissionDate = (new Date(+application.submissionDate)).setHours(0, 0, 0, 0);
```

Comment:
Can we please use JS native `Number()` instead of unary plus symbol?

Reply:
done